app/build.gradle

```diff
@@ -34,6 +34,9 @@ android {
     kotlinOptions {
         jvmTarget = '1.8'
     }
+    //testOptions {
+    //    unitTests.returnDefaultValues = true
+    //}
```

Comment:
I deleted this since I didn't use it

Reply:
I merged with the main branch so it's good now